x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/constants.ts

```diff
@@ -9,5 +9,5 @@ import { i18n } from '@kbn/i18n';
 
 export const CRAWLER_TITLE = i18n.translate(
   'xpack.enterpriseSearch.appSearch.engine.crawler.title',
-  { defaultMessage: 'Crawler' }
+  { defaultMessage: 'Web Crawler' }
```

Comment:
This changes the copy in the navigation link as well - are we good with that? I'm fine either way, maybe would just double check with @daveyholler